web-platform/website/src/components/pages/account-settings/EditAccountPasswordView.tsx

```diff
@@ -0,0 +1,391 @@
+import { FirebaseError } from "@firebase/app";
+import { useRef, useState, useEffect } from "react";
+import {
+  Button,
+  Card,
+  Col,
+  Container,
+  Form,
+  FormFeedback,
+  FormGroup,
+  Input,
+  Label,
+  Row,
+} from "reactstrap";
+import { style } from "typestyle";
+
+import { Strings } from "../../../resources/Strings";
+import { useAuthentication } from "../../../services/AuthenticationService";
+import { Colors, Spacing } from "../../../styles";
+import {
+  DisabledProductButton,
+  ProductButton,
+  TertiaryButton,
+  TransparentButton,
+} from "../../../styles/Buttons";
+import { ContainerStyles } from "../../../styles/ContainerStyles";
+import { FontSizeRaw, Fonts } from "../../../styles/Fonts";
+import { getFirebaseErrorMessage } from "../../../utils/FirebaseErrors";
+import {
+  PasswordAccountValidationResult,
+  validatePasswordAccountForm,
+  validatePasswordRules
+} from "../login/LoginFormValidator";
+import {
+  AccountSettingsState,
+  useAccountSettingsDataContext,
+} from "./AccountSettingsDataContext";
+import { PasswordRules } from "../login/PasswordRules";
+
+const strings = Strings.components.pages.accountSettings.editPasswordAccount;
+const firebaseStings = Strings.utils.firebaseError.errorMessages;
+
+export function EditAccountPasswordView(props:any) {
```

Comment:
We should not be using any as a type. It will actually fail linting and for a good reason. Can we declare a prop type here?